config/routes.rb

```diff
@@ -28,4 +28,5 @@
     resources :orders
   end
 
+  match '/create_order', via: [:get], to: "orders#create"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.